app/controllers/api/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::PostsController < ApplicationController
+  before_action :authenticate_request
+  def index
+    @posts = Post.where(author_id: params[:user_id])
+    if @posts.empty?
+      json_response({ msg: "No Posts found for user #{params[:user_id]}" }, 400)
+    else
+      render json: @posts
```

Comment:
- [ ] Hi @team, I got the error on the screenshot below when I send a request to access all the posts of a particular user.

![Screenshot from 2022-11-11 11-22-15](https://user-images.githubusercontent.com/63087491/201320622-695bae4a-02b6-4283-9bd3-9c330a57912f.png)

Hence I couldn't get comments on a post or create a comment.

- I could get all the users as you can see in the screenshot below:

![Screenshot from 2022-11-11 11-28-13](https://user-images.githubusercontent.com/63087491/201321556-01e9a89e-d8ca-4c6b-b053-fb4068a4be30.png)